samples/bluetooth/hci_uart_async/src/hci_uart_async.c

```diff
@@ -150,7 +150,6 @@ static void send_hw_error(void)
 
 	struct net_buf *buf = bt_buf_get_rx(BT_BUF_EVT, K_FOREVER);
 
-	net_buf_add_u8(buf, BT_HCI_H4_EVT);
```

Comment:
This line needs to stay, I think. I extended the test to cover this here: https://github.com/zephyrproject-rtos/zephyr/pull/89112. (Feel free to cherry-pick it if you like.)

Reply:
> This line needs to stay, I think. I extended the test to cover this here: #89112. (Feel free to cherry-pick it if you like.)

The `bt_buf_get_rx()` function (called just above this) already prepends the appropriate type, so the `add_u8()` would cause a double prefix into the buffer.